src/external_impls.rs

```diff
@@ -43,6 +43,18 @@ mod slab_impl {
     }
 }
 
+#[cfg(feature = "internment")]
+mod internment_impl {
+    use crate::{Context, DeepSizeOf};
+    use internment::Intern;
+
+    impl<T> DeepSizeOf for Intern<T> {
+        fn deep_size_of_children(&self, _context: &mut Context) -> usize {
+            0
```

Comment:
This should probably actually traverse into `T` iff the `Context` shows that it hasn't already been visited (similar to what is done for `Arc`).